internal/guest/prot/protocol.go

```diff
@@ -583,6 +584,12 @@ func UnmarshalContainerModifySettings(b []byte) (*ContainerModifySettings, error
 			return &request, errors.Wrap(err, "failed to unmarshal settings as LCOWSecurityPolicyEnforcer")
 		}
 		msr.Settings = enforcer
+	case guestresource.ResourceTypePolicyFragment:
+		fragment := &ctrdtaskapi.PolicyFragment{}
```

Comment:
I think for consistency sake we should define and pass a `guestresource` type to send fragments over the bridge, rather than re-using the `ctrdtaskapi` type.